punjab/httpb.py

```diff
@@ -615,11 +616,11 @@ def startSession(self, body, xmpp_elements):
         if not body.hasAttribute('rid') or body['rid'] == '':
             if self.v:
                 log.msg('start session called but we had a rid')
-            return None, defer.fail(error.NotFound)
```

Comment:
Why do you change these exceptions?

Reply:
Otherwise `trial xep124.XEP0124TestCase.testWhiteListError` would not pass. But I found a better solution...